msteams-platform/resources/schema/manifest-schema.md

```diff
@@ -198,6 +225,26 @@ Specifies information about your company. For apps submitted to AppSource (forme
 |`websiteUrl`|2048 characters|✔|The https:// URL to the developer's website. This link should take users to your company or product-specific landing page.|
 |`privacyUrl`|2048 characters|✔|The https:// URL to the developer's privacy policy.|
 |`termsOfUseUrl`|2048 characters|✔|The https:// URL to the developer's terms of use.|
+|`mpnId`|10 characters|✔|**Optional** The Microsoft Partner Network ID that identifies the partner organization building the app.|
+
+## localizationInfo
+
+**Optional**
+
+Allows the specification of a default language, as well as pointers to additional language files. See [localization](~/publishing/apps-localization.md).
+
+|Name| Maximum size | Required | Description|
+|---|---|---|---|
+|`defaultLanguageTag`|4 characters|✔|The language tag of the strings in this top level manifest file.|
```

Comment:
technically more characters are allowed. A valid languageTag for us is in the form '8 chars'-'8 chars'-'8 chars', however this doesn't make sense for most languages.

Reply:
fixed (all 3)